[2022-01-25 11:04:08,529] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T10:57:24.583418+00:00 [queued]>
[2022-01-25 11:04:08,621] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: classification.load_data 2022-01-25T10:57:24.583418+00:00 [queued]>
[2022-01-25 11:04:08,624] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 11:04:08,628] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-01-25 11:04:08,628] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-01-25 11:04:08,670] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): load_data> on 2022-01-25T10:57:24.583418+00:00
[2022-01-25 11:04:08,687] {standard_task_runner.py:52} INFO - Started process 11858 to run task
[2022-01-25 11:04:08,729] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'classification', 'load_data', '2022-01-25T10:57:24.583418+00:00', '--job-id', '109', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/classification.py', '--cfg-path', '/tmp/tmpu2nqoen5', '--error-file', '/tmp/tmpl5evj3h9']
[2022-01-25 11:04:08,748] {standard_task_runner.py:77} INFO - Job 109: Subtask load_data
[2022-01-25 11:04:08,836] {logging_mixin.py:104} INFO - Running <TaskInstance: classification.load_data 2022-01-25T10:57:24.583418+00:00 [running]> on host 54a54d544bf0
[2022-01-25 11:04:08,921] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=classification
AIRFLOW_CTX_TASK_ID=load_data
AIRFLOW_CTX_EXECUTION_DATE=2022-01-25T10:57:24.583418+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-25T10:57:24.583418+00:00
[2022-01-25 11:04:49,176] {python.py:118} INFO - Done. Returned value was: None
[2022-01-25 11:04:49,247] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=classification, task_id=load_data, execution_date=20220125T105724, start_date=20220125T110408, end_date=20220125T110449
[2022-01-25 11:04:49,536] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-01-25 11:04:49,635] {local_task_job.py:146} INFO - Task exited with return code 0
